libs/shared/contentful/src/lib/contentful.manager.ts

```diff
@@ -79,6 +84,22 @@ export class ContentfulManager {
     );
   }
 
+  async getPageContentForOffering(
+    offering: string,
```

Comment:
Small change, could this be a updated to be more specific?

I had to look at the actual graphql query to realize this value is used for the Offering.apiIdentifer. Ideally I'd be able to tell just by looking at this method.